src/main/java/com/epam/wl/dao/UserDAO.java

```diff
@@ -66,5 +70,9 @@ public void deleteUserById(int id) throws SQLException {
     public Optional<User> getUserByNameAndLastName(String name, String lastName) throws SQLException {
         return executor.executeQuery(GET_USER_SCRIPT_By_NAME_LASTRNAME_SCRIPT, userOneHandler, name, lastName);
     }
+
+    public boolean isLibrarian(String email, String password) throws SQLException {
+        return executor.executeQuery(IS_LIBRARIAN_QUERY, userIsLibrarianHandler, email, password);
```

Comment:
I think it's wrong to put logic about login (checking are username and password correct) in method isLibrarian